app/assets/v2/css/labs.css

```diff
@@ -0,0 +1,183 @@
+html {
+  overflow: hidden;
+}
+
+body {
+  height: 100vh;
```

Comment:
Why overflow hidden and height 100vh is needed? the page was scrolling horizontal or something like that? 
I would like to avoid this if is possible

Reply:
Yes, this is for allowing page to scroll only vertically and is needed.

Reply:
Fixed!